services/web-app/data/libraries.js

```diff
@@ -97,6 +97,13 @@ const prodLibraries = {
     sponsor: 'carbon',
     group: carbonComponentsGroup
   },
+  'carbon-components-vue': {
+    host: 'github.com',
+    org: 'carbon-design-system',
+    repo: 'carbon-components-vue',
+    path: '/packages/core/src',
+    group: carbonComponentsGroup
+  },
```

Comment:
Must specify packageJsonPath since it's not at the same level
```suggestion
  'carbon-components-vue': {
    host: 'github.com',
    org: 'carbon-design-system',
    repo: 'carbon-components-vue',
    path: '/packages/core/src',
    packageJsonPath: '/../package.json',
    group: carbonComponentsGroup
  },
```